src/review/views.py

```diff
@@ -509,28 +507,30 @@ def form_valid(self, form):
         return super(ReviewerDecisionEmail, self).form_valid(form)
 
     def get_success_url(self):
-
-        if self.access_key:
-            return reverse(
-                'review_with_access_key',
-                kwargs={
-                    'review_type': self.review_type,
-                    'submission_id': self.submission.pk,
-                    'access_key': self.access_key,
-                    'review_round':
-                        self.review_assignment.review_round.round_number
-                }
-            )
+        if self.decision == 'accept':
+            if self.access_key:
```

Comment:
There seems to be a bit of overlapping code between this `if/else`, so just a suggestion; something like:
```
if self.decision == 'accept':
    kwargs={
        'review_type': self.review_type,
        'submission_id': self.submission.pk,
        'review_round':
            self.review_assignment.review_round.round_number
    }
    if self.access_key:
        url_name = 'review_with_access_key'
        kwargs['access_key'] = self.access_key
    else:
        url_name = 'review_without_access_key'
    
    return reverse(
        url_name,
        kwargs
    )
```

Reply:
Thanks for this - always keen to cut down on repetition.